resources/sushy-tools/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+
+RUN apt update && apt install -y libvirt-dev && cd /root/ && \
+    git clone https://opendev.org/openstack/sushy-tools.git && \
+    cd /root/sushy-tools && \
+    pip3 install -r requirements.txt && python3 setup.py build && \
```

Comment:
Why not install `sushy-tools` via pip?

Reply:
if we install sushy-tools via pip, the managers are not implemented (pip3 install sushy-tools==0.6.0). We need this feature, hence the installation from source

Reply:
I'll request a release.

Reply:
Could you please wait for the release request ? we would need another fix in : https://review.opendev.org/#/c/677449/

Reply:
yeah, the release was delayed because of other outstanding issues